kafka_consumer/datadog_checks/kafka_consumer/legacy_0_10_2.py

```diff
@@ -221,6 +223,9 @@ def _get_highwater_offsets(self):
             )
             response = self._make_blocking_req(request, node_id=broker.nodeId)
             self._process_highwater_offsets(response)
+            if len(self._highwater_offsets) >= contexts_limit:
+                self.log.debug("Context limit reached, not collecting more highwater offsets")
```

Comment:
```suggestion
                self.log.debug("Context limit reached. Skipping highwater offsets collection.")
```

match log line from new version